install.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+# XXX: Update to final destination before merge:
+# SCAF_SCRIPT_URL="https://raw.githubusercontent.com/sixfeetup/scaf/main/scaf"
+SCAF_SCRIPT_URL="https://raw.githubusercontent.com/sixfeetup/cookiecutter-sixiedjango/rcompaan/157-simplify-install/scaf"
+TEMP_DOWNLOAD="./scaf"
+DESTINATION="/usr/local/bin/scaf"
+
+command_exists() {
+    type "$1" &> /dev/null
+}
+
+prompt_install() {
+    read -p "Do you want to install $1? (y/n) " answer
+    if [[ $answer = [Yy]* ]]; then
+        install_$1
+    else
+        echo "Skipping installation of $1."
+    fi
+}
+
+detect_os_and_arch() {
+    os=$(uname -s)
+    arch=$(uname -m)
+
+    case "$os" in
+        Linux*)
+            case "$arch" in
+                x86_64) os_arch="linux-amd64" ;;
+                aarch64) os_arch="linux-arm64" ;;
+                *)      os_arch="linux-unknown" ;;
+            esac
+            ;;
+        Darwin*)
+            case "$arch" in
+                x86_64) os_arch="darwin-amd64" ;;
+                arm64)  os_arch="darwin-arm64" ;;
+                *)      os_arch="darwin-unknown" ;;
+            esac
+            ;;
+        *)          
+            os_arch="unknown"
+            ;;
+    esac
+    echo $os_arch
+}
+
+install_kubectl() {
+    echo "Installing kubectl..."
+    os_arch=$(detect_os_and_arch)
+
+    case $os_arch in
+        "linux-amd64"|"darwin-amd64"|"darwin-arm64"|"linux-arm64")
+            url="https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/$os_arch/kubectl"
```

Comment:
I figured out why kubectl install was failing.
This url gets rendered as: ` https://dl.k8s.io/release/v1.29.0/bin/linux-amd64/kubectl` but the correct CDN path is:
`https://dl.k8s.io/release/v1.29.0/bin/linux/amd64/kubectl`
The output from curl is the cdn's "key-not-found" error message which gets stored in `/usr/local/bin/kubectl`